### ISSUE TITLE
🐛 

### ISSUE THREAD

**Issue by BoBeR182** [ookangzheng/blahdns#214]
**Checklist**

- [X] I read the [README](https://github.com/ookangzheng/blahdns/blob/master/README.md)
- [X] I read the [FAQ](https://github.com/ookangzheng/blahdns/blob/master/FAQ.md)
- [X] I searched the [issues](https://github.com/ookangzheng/blahdns/issues?q=is%3Aissue+)

**Describe the issue**
A clear and concise description of what the issue is.

orpheus.network does not resolve

**To Reproduce**

resolvectl query orpheus.network                
orpheus.network: resolve call failed: Received invalid reply

**Logs**
Global
           Protocols: +LLMNR +mDNS +DNSOverTLS DNSSEC=no/unsupported
    resolv.conf mode: stub
  Current DNS Server: 78.46.244.143#dot-de.blahdns.com
         DNS Servers: 2a01:4f8:c17:ec67::1#dot-de.blahdns.com 78.46.244.143#dot-de.blahdns.com
Fallback DNS Servers: 1.1.1.1 9.9.9.10 8.8.8.8 2606:4700:4700::1111 2620:fe::10 2001:4860:4860::8888
          DNS Domain: ~.

Link 2 (enp3s0)
Current Scopes: LLMNR/IPv4 LLMNR/IPv6
     Protocols: -DefaultRoute +LLMNR -mDNS +DNSOverTLS DNSSEC=no/unsupported

**Expected behavior**
Return IPv4 178.175.138.98

**Client (if applicable):**

- Device: Linux Desktop
- OS: Arch
- Client: resolvectl
- Protocol: DoT

**Comment by ookangzheng** [comment]
domain `orpheus.network` can be resolved via DoT on `dot-de.blahdns.com`

```
kdig orpheus.network @dot-de.blahdns.com +tls
;; TLS session (TLS1.3)-(ECDHE-X25519)-(RSA-PSS-RSAE-SHA256)-(AES-256-GCM)
;; ->>HEADER<<- opcode: QUERY; status: NOERROR; id: 28770
;; Flags: qr rd ra ad; QUERY: 1; ANSWER: 1; AUTHORITY: 0; ADDITIONAL: 1

;; EDNS PSEUDOSECTION:
;; Version: 0; flags: do; UDP size: 4096 B; ext-rcode: NOERROR
;; PADDING: 389 B

;; QUESTION SECTION:
;; orpheus.network.    		IN	A

;; ANSWER SECTION:
orpheus.network.    	3600	IN	A	178.175.138.98

;; Received 468 B
;; Time 2021-09-10 06:31:53 UTC
;; From 2a01:4f8:c17:ec67::1@853(TCP) in 568.3 ms
```

**Comment by BoBeR182** [comment]
; <<>> DiG 9.16.20 <<>> orpheus.network
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: SERVFAIL, id: 26804
;; flags: qr rd ra; QUERY: 1, ANSWER: 0, AUTHORITY: 0, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 65494
;; QUESTION SECTION:
;orpheus.network.		IN	A

;; Query time: 323 msec
;; SERVER: 127.0.0.53#53(127.0.0.53)
;; WHEN: Fri Sep 10 02:36:21 EDT 2021
;; MSG SIZE  rcvd: 44

**Comment by ookangzheng** [comment]
Still not working?

**Comment by BoBeR182** [comment]
Correct, is there any additional trouble shooting steps I can provide for debugging?
